compiler/testData/codegen/bytecodeText/constants/comparisonFalse.kt

```diff
@@ -0,0 +1,22 @@
+// TARGET_BACKEND: JVM_IR
```

Comment:
@ting-yuan Please add box tests for  comparisonFalse/comparisonTrue

Reply:
Done!